lib/resty/redis.lua

```diff
@@ -20,67 +20,25 @@ if not ok or type(new_tab) ~= "function" then
 end
 
 
-local _M = new_tab(0, 155)
+-- #commands(36) + #sub_commands(2) + #unsub_commands(2)
+-- + others(24) + reservation(10) = 64
+local _M = new_tab(0, 64)
 _M._VERSION = '0.24'
 
 
 local commands = {
```

Comment:
Hmm, maybe we can change the variable name from `commands` to `common_cmds` to avoid people from trying to add entries to this array :)